### PR TITLE
Register luisalejandrobf

### PR DESCRIPTION
**Pull request by luisalejandrobf**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview
[https://luisalejandrobf.vercel.app/](https://luisalejandrobf.vercel.app/)

### CODE REVIEW

File: domains/luisalejandrobf.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "luisalejandrobf",
+    "email": "luisalejandrobravoferreira@gmail.com"
+  },
+  "record": {
+    "TXT": "google-site-verification=sGmOAXQM9W1XepIo-CgWM_GlqEMW7z96P10kOC2p6fs"
```
Comment (CuteDog5695) [inline domains/luisalejandrobf.json:7]: ?
Please read the docs on how to use Vercel with the domain https://is-a.dev/docs/vercel/

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/luisalejandrobf.json (hunk 1 above)
Comment (luisalejandrobf) [reply domains/luisalejandrobf.json:7]: Okay, sorry, I didn't know. I just read the docs, but I wanted to ask: If I use the CNAME or A record methods, will I be able to include a TXT record so I can try and optimize SEO a little with google search console? Thanks!

Comment (CuteDog5695) [reply domains/luisalejandrobf.json:8]: Yes, should be fine if you are using it with A records. With CNAME no since you can't add other records with it

**Review by iostpa — APPROVED** [review]
Seems good to me now

File: domains/luisalejandrobf.json (hunk 1 above)
Comment (luisalejandrobf) [reply domains/luisalejandrobf.json:8]: Sure, thank you

**Pull request merged**